auth_totp/wizards/res_users_authenticator_create.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+import urllib
+from openerp import api, fields, models
+from openerp.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+try:
+    import pyotp
+except ImportError:
+    _logger.debug(
+        'Could not import PyOTP. Please make sure this library is available in'
+        ' your environment.'
+    )
+
+
+class ResUsersAuthenticatorCreate(models.TransientModel):
+    _name = 'res.users.authenticator.create'
+    _description = 'MFA App/Device Creation Wizard'
+
+    name = fields.Char(
+        string='Authentication App/Device Name',
+        help='A name that will help you remember this authentication'
+        ' app/device',
+        required=True,
+        index=True,
+    )
+    secret_key = fields.Char(
+        default=lambda s: pyotp.random_base32(),
```

Comment:
If our data models require it, we should always require it. Considering what is or is not exposed in a view is out of scope for a model.